python/whylogs/core/view/dataset_profile_view.py

```diff
@@ -230,6 +230,6 @@ def to_pandas(self, column_metric: Optional[str] = None, cfg: Optional[SummaryCo
             sum_dict = col.to_summary_dict(column_metric=column_metric, cfg=cfg)
             sum_dict["column"] = col_name
             sum_dict["type"] = SummaryType.COLUMN
-            all_dicts.append(sum_dict)
+            all_dicts.append(dict(sorted(sum_dict.items())))
```

Comment:
Ahh, that makes sense, so maybe we need to sort at both locations? I'm not too worried about the performance of sorting inside the loop I was concerned it might not be sufficient. The to_pandas() callstack isn't part of the performance critical code of profiling and is a callstack that I expect to be done less frequently as part of analyzing the profiles after logging is done.